themes/storefront/class-hfe-storefront-compat.php

```diff
@@ -13,14 +13,17 @@ class HFE_Storefront_Compat {
 	/**
 	 * Instance of HFE_Storefront_Compat.
 	 *
-	 * @var HFE_Storefront_Compat
+	 * @var ?HFE_Storefront_Compat
```

Comment:
Spell mistake use `$` instead of `?`
```suggestion
	 * @var $HFE_Storefront_Compat
```

Reply:
This is updated @sarangshahane